contrib/terraform/techdocs-s3-storage/terraform.tf

```diff
@@ -0,0 +1,81 @@
+#==========================
+# Variables
+#==========================
+
+variable "backstage" {
+  default = "backstage_bucket_for_my_corp"
+}
+
+variable "backstage-iam" {
```

Comment:
`backstage_iam` & `shared_mamahed_tag_value`?

Reply:
@JacobValdemar ping here too

Reply:
@JacobValdemar there's this little change left sorry :)

Reply:
@benjdlambert Can you please elaborate on what's left? 🙂

Reply:
Sure - just to change the var names to be "_" instead of "-"

Makes for better reading and writing.

`aws_iam_role.backstage_user` is nicer than `aws_iam_role.backstage-thing`

Reply:
Also this: `data.aws_iam_policy_document.backstage-policy.json`